src/dappsExplorer/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { DappV1, DappV2, isDappV2 } from 'src/dapps/types'
+
+export const calculateSearchScore = (dapp: DappV2 | DappV1, searchQuery: string) => {
```

Comment:
if i understand this correctly, there are some rules here to order the search results depending on how much they "match" the search results. is that right? i don't remember talking about anything like this at refinement, is this kind of logic necessary for this experiment? can we just do a black and white search, where we display all matched dapps in the existing order form the backend?

tbh i feel like this kind of ranking logic shouldn't be on the client :/ i'm not sure exactly where this should go, but it's feasible to imagine a world where we decide next month that actually we want to change this logic completely (e.g. we no longer care at all about matching the category) and would want this change to affect every app version. if this logic is baked into apps, then we don't have control anymore and we're bound to the release process for changes. 

unless this logic is very important to the experiment, i'd suggest leaving it out. if it is important, we should maybe have a quick brainstorm and weigh up pros / cons of various ways to implement this.